pages/linux/pacdiff.md

```diff
@@ -0,0 +1,32 @@
+# pacdiff
+
+> Pacorig, pacnew and pacsave maintenance utility for pacman.
+> More information: <https://manned.org/pacdiff>.
+
+- Look for files that need maintenance:
+
+`pacdiff`
+
+- Use sudo and sudoedit to remove and merge files:
+
+`pacdiff --sudo`
+
+- Store the original version of any file you `(O)verwrite with` as a `.bak`:
+
+`pacdiff --backup`
+
+- Override the program for `(M)erge`ing changes (`vim -d` is default):
```

Comment:
Similar comment as above, using `\`(M)erge\`` is special syntax to indicate that a concept maps to a CLI flag. Is this your intent?